src/command_modules/azure-cli-deploymentmanager/azure/cli/command_modules/deploymentmanager/_help.py

```diff
@@ -0,0 +1,198 @@
+"""
+
+helps['deploymentmanager service create'] = """
+    type: command
+    short-summary: Creates a service under the specified service topology.
+    examples:
+        - name: Create a new service under a service topology. Specify the service by its name, service topology it is in and the resource group name.
+          text: >
+            az deploymentmanager service create -g rg1 -location centralus --service-topology-name contosoServiceTopology --service-name contosoService1 --target-location "East US" --target-subscription-id XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+"""
+
+helps['deploymentmanager service show'] = """
+    type: command
+    short-summary: Get the details of a service.
+    examples:
+        - name: Get the service under a service topology.
+          text: >
+            az deploymentmanager service show -g rg1 --service-topology-name contosoServiceTopology --service-name contosoService1
+"""
+
+helps['deploymentmanager service delete'] = """
+    type: command
+    short-summary: Deletes the service topology.
+    examples:
+        - name: Deletes a service topology.
+          text: >
+            az deploymentmanager service delete -g rg1 --service-topology-name contosoServiceTopology --service-name contosoService1
+"""
+
+helps['deploymentmanager service update'] = """
+    type: command
+    short-summary: Updates the service.
+    examples:
+        - name: Updates the service.
+          text: >
+            az deploymentmanager service update -g rg1 --service-topology-name contosoServiceTopology --service-name contosoService1 --target-location "West US" --target-subscription-id XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+"""
+
+helps['deploymentmanager service-unit create'] = """
+    type: command
+    short-summary: Creates a service unit under the specified service and service topology.
+    examples:
+        - name: Create a new service unit using relative paths into the artifact source.
+          description:
+            Specify the service unit by its name, the service and service topology it is in. 
+            The template and parameters files are defined as relative paths into the artifact source location referenced in the specified service topology.
+            The resources defined in this template are to be deployed into the target resource group service1ResourceGroup with the deployment mode set to 'Incremental'.
+          text: >
+            az deploymentmanager service-unit create -g rg1 -location centralus --service-topology-name contosoServiceTopology --service-name contosoService1 --service-unit-name ContosoService1Storage --target-resource-group service1ResourceGroup --deployment-mode Incremental --template-artifact-source-relative-path "Templates/Service1.Storage.json" --parameters-artifact-source-relative-path "Parameters/Service1.Storage.Parameters.json" 
```

Comment:
This argument `--template-artifact-source-relative-path` should just be named `template-file` see the usage for the deployment create command:
https://github.com/Azure/azure-cli/blob/ada2a74c57346a9e587b595c653ce91f0f67712a/src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py#L214-L238

Reply:
There are two template related parameters for this cmd. One is --template-uri that is consistent with how 'deployment create' works, it takes a sas uri to the template file. The second --template-artifact-source-relative-path is not the same as 'template-file'. template-file refers to the path to the file on the local machine. 'template-artifact-source-relative-path' is a path relative to the artifact-source that the users have already registered with deployment manager. You can refer to this here: 

https://docs.microsoft.com/en-us/azure/templates/Microsoft.DeploymentManager/2018-09-01-preview/serviceTopologies/services/serviceUnits

https://docs.microsoft.com/en-us/azure/azure-resource-manager/deployment-manager-overview#service-units

Reply:
After our discussion, combining the path args, one for template and one for params file.